src/transaction/atomic_replication_helper.cpp

```diff
@@ -533,32 +628,15 @@ namespace cublog
       case RVHF_UPDATE:
       case RVHF_MVCC_UPDATE_OVERFLOW:
       case RVHF_INSERT_NEWHOME:
-	// sanity asserts inside the function
-	pgbuf_ordered_unfix (thread_p, m_watcher_p.get ());
+	assert (page_ptr == nullptr);
```

Comment:
I think `RVHF_UPDATE_NOTIFY_VACUUM` should be added as a case here.
`RVHF_UPDATE_NOTIFY_VACUUM` is a recovery index for mvcc update. 
```suggestion
      case RVHF_UPDATE_NOTIFY_VACUUM:
	assert (page_ptr == nullptr);
```

Reply:
Related issue : http://jira.cubrid.org/browse/LETS-422

Reply:
Done.